app/assets/javascripts/modules/remove-filter.js

```diff
@@ -0,0 +1,51 @@
+window.GOVUK = window.GOVUK || {};
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (global, GOVUK) {
+  'use strict';
+
+  GOVUK.Modules.RemoveFilter = function () {
+    this.start = function (element) {
+      $(element).on('click', '[data-module="remove-filter-link"]', toggleFilter);
+    };
+
+    function toggleFilter(e) {
+      e.preventDefault();
+
+      var removeFilterName = $(this).data("name");
+      var removeFilterValue = $(this).data("value");
+      var removeFilterFacet = $(this).data("facet");
```

Comment:
Just noticed here we're using `"` but in other lines we're using `'` - sorry, small thing, shouldn't block the PR.